operator/src/main/java/org/bf2/operator/operands/KafkaInstance.java

```diff
@@ -47,6 +54,19 @@ void init() {
 
     @Override
     public void createOrUpdate(ManagedKafka managedKafka) {
+        Tags tags = MetricsManager.buildKafkaInstanceTags(managedKafka);
+
+        if (managedKafka.getAnnotation(ManagedKafkaKeys.Annotations.PAUSE_RECONCILIATION).map(Boolean::valueOf).orElse(false)) {
+            meterRegistry.gauge("kafka_instance_paused", tags, 1);
```

Comment:
small nit, maybe we can pull a constant for a string in MetricsManager?

Reply:
Done